app/assets/stylesheets/tooltip.css.scss

```diff
@@ -0,0 +1,75 @@
+
+.tooltip-item {  
+  ///////////////////////////////////////////////////////////////////////////////////
```

Comment:
Rule sets should be ordered as follows: @extends, @includes without @content, properties, @includes with @content, nested rule sets